base/uk.ac.stfc.isis.ibex.experimentdetails/src/uk/ac/stfc/isis/ibex/experimentdetails/UserDetails.java

```diff
@@ -73,31 +73,31 @@ public UserDetails(String name, String institute, Role role) {
      * @return The user's full name.
      */
 	public String getName() {
-		return name;
+		return name.replaceAll("^\\s+|\\s+$", "");
```

Comment:
I think you should use `.trim()` rather than a regular expression; this will be more robust and clearer as to what it's doing

Reply:
I was originally going to use trim but the ticket asked for something that would get rid of any white space, and I think trim only removes spaces and not any white space. However, I don't think it's possible to enter from the GUI anything other than a space as white space in the different names, so I don't know if the requirement to remove any white space is justified...

Reply:
`trim` does get rid of any whitespace

Reply:
Ok thanks!